contrastsecurity/assets/screen/contrast_security_protect.json

```diff
@@ -0,0 +1,285 @@
+{
+  "title": "Contrast Security - Protect",
```

Comment:
how has this json been generated? It seems to be missing some fields such as author_info and description

Reply:
this is our json that we load in for the dashboard to work with our product. I PR'd in integrations-extra and they told me that in order to create a button that automatically downloads the json into the users dashboard that I needed to PR here and add my JSON to assets folder. Is that correct?

Reply:
What I meant [here](https://github.com/DataDog/integrations-extras/pull/503/files#r349581063) is is that your JSON file should follow the *format* of the ones you can find on this repo not to move the file here. There are different versions and classes of dashboards. Integrations default dashboards should follow a specific format.